Add Up-to-Date Work

MatthewChastain opened the pull request:
This PR adds in all work on the DASSL Duck project up to the end of the Spring 2019 semester. The `DASSLduck.obj` file was added and overwritten with each successive version to keep the version control scheme intact. Images of past versions of the duck are in the images/ directory in specified folders for each past version. The README.md has also been updated to reflect the changes made to the repo.

Commit:
Update README.md

README.md now informs reader about the dassl Duck project.

Commit:
Update README.md

Commit:
Upload v1

Commit:
Upload v2

Commit:
Upload v3

Commit:
Upload v4

Commit:
Upload v5

Commit:
Adding gcode and Updating README.md

Commit:
Fix Lettercase

afig reviewed (commented):
Thanks for adding these files and images @MatthewChastain.

Really, the only suggestion I have is to perhaps provide the approximate size of the ducks created by the `.gcode` files. These sizes can be added somewhere in the readme and/or in the files names. If the sizes are in the readme, it might be sufficient to only give the size for the normal and XXXL sized ducks.

Due to the images and `.gcode` files, the repository is quite large, but I think it is okay since it won't be modified too frequently.

Commit:
Make purpose apply to all; not just students

Edit opening paragraph to remove the limitation on the duck assisting only students.

Commit:
Add code markdown

Add code markdown to the last paragraph and reword a bit

smurthys commented:
Fantastic work @MatthewChastain. Thank you for the contribution. 

I confirm I am able to open the `obj` file in `Paint 3D` on Windows, but that is all I am able to verify at this point. I did make a couple of very minor edits to `README`, and agree with @afig's suggestions to add size information. Other than the size information, I feel these changes are ready to merge.

**The following suggestion/desire is for another PR:** It will be very helpful, if a "how to" document is added to provide a sequence of DIY steps to create a finished duck. This doc has two parts: the first part is includes the sequence of operations in various software tools to end up with a .obj file. The second part is the operations performed with a printed duck to produce a finished duck. I imagine especially the second part of the doc to take the form of DIY recipes commonly seen on the web where they show the result of each step so followers can verify result at each stage. More important, I think this doc will help the reader fully appreciate the amazing organization, effort, and discipline you have put into producing this artifact.

smurthys reviewed (approved):
Ready to merge 👍

MatthewChastain committed:
Adding Sizes of Generated Gcode

Included the sizes of printed models from each .gcode file in the `gcode/` directory.

MatthewChastain committed:
Update README.md

MatthewChastain committed:
Update README.md

MatthewChastain committed:
Update README.md

MatthewChastain commented:
I've added in the sizes of each resulting model from the .gcode files into the `README.md` but I don't think the table is rendering properly from the markdown, perhaps I'm missing something?

afig commented on README.md:
```diff
@@ -1,2 +1,35 @@
+This repository contains the current version of the DASSL Duck in a .obj 3D file format which should be compatible for any slicing engine. For convenience there is also the `gcode/` directory that contains pre-compiled gcode for use on most standard FDM printers. There are multiple different sizes of the model with `DASSLduck.obj` being the standard size. The repository also contains an `images/` ...
+
+## Gcode Sizes
+These are the sizes of the models that will be printed using the generated gcode files in the `gcode/` directory:  
```
```suggestion
These are the sizes of the models that will be printed using the generated gcode files in the `gcode/` directory:

```

afig reviewed (commented):
Thanks for adding the sizes @MatthewChastain. Looks like the issue with the tables is caused by there needing to be a blank line before the table.

afig reviewed (approved):
Good to merge after the suggested change has been accepted or otherwise addressed

MatthewChastain committed:
Update README.md

Co-Authored-By: Andrew Figueroa <afig23@gmail.com>
